docs/bes-open-source-security-tool-tech-stack.md

```diff
@@ -10,7 +10,7 @@ hide:
 
 <h1> Open Source Security Tools [S] Tech Stack </h1>
```

Comment:
Please raise a pull from base repo first. I have made some changes in this file.

Reply:
@asa1997 
Its already done, pull request is from synced feature branch (fb-3).